.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.17.8
```

Comment:
I'd argue we don't want to add such specific dotfiles to the drand repo itself. 
Maybe add this file to our `.gitignore` file?

Reply:
This is to ensure that everyone uses the same version of Go. I think that it increases stability and reproducibility (as long as everyone uses `asdf` of course). I would even move to have CI use `asdf` too.

Reply:
We currently set the expected version of golang in the docker file and CI. I'd prefer to not add a dependency on `asdf` since none of the development team is using it.

Reply:
OK, I've removed the file and added it to `.gitignore`.